test/ref/continuous_test.ref.json

```diff
@@ -879,7 +879,7 @@
   ]
 },
 {
-  "expr": "Erlang(3)",
+  "expr": "Erlang(alpha=3)",
```

Comment:
are you manually editing these, or were they generated from the R script?

Reply:
These are supposed to be generated by the `gendref.R` file.

Reply:
ok, but the `.lst` is supposed to contain the Julia syntax correct?

Reply:
Honestly, I didn't try to figure it out.

Reply:
I'm getting the R deps now to re-generate the JSON and see if all good